.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
           name: Install Dependencies
           command: |
             go get -u github.com/golang/dep/cmd/dep
-            dep ensure
+            dep ensure -vendor-only
```

Comment:
Not sure this is what we want here as it won't add any updates to the lock file when we deploy. I would actually prefer to run `dep ensure -update` here to break the build if we have failed tests will all updated libraries.

Reply:
I don't want to add any updates to the lock file. I want to deploy exactly what we have developed on.